UnrealPlugin/README.md

```diff
@@ -10,337 +11,38 @@ It was tested with Unreal Engine 4.26.2, but should work with other engines as w
-
-Don't forget to connect all the nodes to the Event Init node.
-
-In the end add the "Ready for Players" to be able to react to the ready signal of PlayFab.
-![PlayFab GSDK Full Graph](Documentation/BlueprintFullGraph.png)
-
-## Set the Game Instance class
-    
-After creating a custom game instance class that integrates with the gsdk, you have to configure your project to actually use this newly created game instance class. There are two ways to do this - either through the Unreal Engine editor or by editing DefaultEngine.ini directly.
-
-###### In the Unreal Editor
-In the editor, this can also be set through the UI in the editor. In the editor go to Edit -> Project Settings. From that opened window,
-navigate to Maps&Modes on the left side. Scroll to the bottom, and then you can set the option "Game Instance Class" to your new game instance class directly, and avoid typos.
-
-###### In DefaultEngine.ini
-Or you can update DefaultEngine.ini file and add this:
-```ini
-[/Script/EngineSettings.GameMapsSettings]
-GameInstanceClass=/Script/[game name].MyGameInstance
-```
-
-## Include Pre-requisites for Windows Dedicated Server
-There are two ways to include the app-local prerequisites - either through the Unreal Engine editor or by editing DefaultGame.ini.
-
-### In the Unreal Editor
-In the editor go to Edit -> Project Settings. In the opened window navigate to Packaging on the left side. Scroll to the bottom of the list, and tick "Include app-local prerequisites".
-
-### In DefaultGame.ini
-Or you can update DefaultGame.ini to show the following:
-```ini
-[/Script/UnrealEd.ProjectPackagingSettings]
-IncludeAppLocalPrerequisites=True
-```
-
-If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
-
-If you are using Continuous Integration (CI), then you could add it to your setup to only turn this flag on when building a dedicated server, so the additional dlls only get added if it is a dedicated server build.
-
-## Packaging the game server
-
-Make sure that you have just built your project in development editor in Visual Studio *before* doing this next step, because sometimes building fails when opening the project directly through the .uproject file.
+## Requirements
 
-Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files". 
+* Download Visual Studio (the [community version is free](https://visualstudio.microsoft.com/vs/community/))
+	- Requires Workloads: .NET desktop development, and Desktop development with C++
+* Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal Engine website. This was tested on [4.26.2](https://github.com/EpicGames/UnrealEngine/releases/tag/4.26.2-release) (Link requires permissions)
```

Comment:
Download Unreal Engine Source Code. This plugin was tested on Unreal Engine 4.26.2. For instructions, see [Downloading Unreal Engine Source code (external)] (https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/).